packages/sui-js/src/cookie/index.js

```diff
@@ -0,0 +1,14 @@
+import {parse as parseCookie} from 'cookie'
+import jsCookie from 'js-cookie'
```

Comment:
I know that i had this conversation. Need we have this two libraries ?!

Reply:
Not sure about that. i think this two libraries has different features. i guess we can look for another one which includes all features we need.

Reply:
i could not find a repo with all features we need. i guess that's because parse is a different concept than get and set (even if they are closely related).
let's work with that!